src/main/scala/org/allenai/plugins/DockerBuildPlugin.scala

```diff
@@ -0,0 +1,135 @@
+package org.allenai.plugins
+
+import sbt.{
+  relativeTo,
+  settingKey,
+  taskKey,
+  AutoPlugin,
+  Def,
+  Keys,
+  PathFinder,
+  Plugins,
+  SettingKey,
+  TaskKey
+}
+import sbt.plugins.JvmPlugin
+
+import java.io.File
+
+/** Plugin for building docker images. */
+object DockerBuildPlugin extends AutoPlugin {
+  val AI2_PRIVATE_REGISTRY = "allenai-docker-private-docker.bintray.io"
+
+  val DEFAULT_BASE_IMAGE = AI2_PRIVATE_REGISTRY + "/java:8"
```

Comment:
This is nominally customized (really, just in that it uses a specific ubuntu version).

Since the registry is part of the image name, it seemed fine to me to use `java` directly. I mean, we could do `allenai/java`, which I initially had, but that seems weird with the registry prefix.

There's nothing stopping us from making this be `allenai/java` if it were ever to be released into the wild on Docker hub.